tests/codegen/waiter-tests/model/waiter-operations.smithy

```diff
@@ -193,6 +193,50 @@ service WaitersTestService {
         ]
     },
 
+    // list indexing
```

Comment:
**suggestion**: Would be good to also test a more involved index expression like the struct list `list.structs[1].strings[0]`